core/db/migrate/20140309033438_create_store_from_preferences.rb

```diff
@@ -1,5 +1,12 @@
 class CreateStoreFromPreferences < ActiveRecord::Migration
   def change
+    # workaround for spree_i18n and Store translations
+    Spree::Store.class_eval do
+      def self.translated?(name)
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `translated?(*)` if you want the method to accept any arguments but don't care about them.